libx52/x52_control.c

```diff
@@ -22,7 +22,7 @@
 int libx52_vendor_command(libx52_device *x52, uint16_t index, uint16_t value)
 {
     int j;
-    int rc;
+    int rc = LIBUSB_ERROR_OTHER;
```

Comment:
You're right. I'll revert this one. This warning shows up with -Wconditional-uninitialized which is on by default in Xcode but can have false positives.